active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
+- Members can tag,label and respond to issues.
+- Member can maintain and monitor the website including updating versions
+- Mentor new contributors to the website
+- Chair, Co-Chair and Board Liaison can sign off on new features
+
+## Initial membership
+
+- Chair: TBA
+- Co-Chair: TBA
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBA
+- Other members:
+    - Sarah Abderemane
+    - Eric Sherman
+    - Mark Walker
+    - Jason Judkins
+    - Paolo Melchiorre
+    - Sanyam Khurana
+    - Tobias McNulty
+    - Ron Maravanyika
+
+
+
+## Future membership
+
+### Who is eligible to join? Any volunteer, or are there specific requirements?
+
+Members must have interest in Django and should be able to work with Django. Most importantly members must be willing to adhere to Django's [Code of Conduct](https://www.djangoproject.com/conduct/) . Members must be well versed with the process of contributing to **djangoproject.com** or at least willing to be guide. We welcome all experience levels, we also welcome first time contributors. 
+
+### How do people who want to join sign up / volunteer / express interest?
+Individuals can express interest by emailing to the working group mailing list at `website-wg@djangoproject.com` 
```

Comment:
Could we set up a Google Form instead? From my experience with DSF mailing lists, it’s 99% spam that we receive on those emails.

It’d be nice for you to define here what kind of information people should provide when applying, rather than just expect applicants will figure it out.

Reply:
@django/djangoproject-com-maintainters what questions must we ask if we are to build the form?

Reply:
I’d recommend "has to be a DSF member" as this is a clearly-defined status.

Reply:
I second the suggestion to be a DSF member to join.

Reply:
I agree with "has to be a DSF member"

Reply:
@thibaudcolas I discovered yesterday during the WG conversations that I am *not* yet a DSF member, but I have now been nominated. If that nomination goes through, and if this proposal is not yet finalized by that time, could you add me to the initial Working Group membership, please? Thank you!

Reply:
> Members must be well versed with the process of contributing to **djangoproject.com**

So, this rules out new people - people who might be excited about helping but don't have experience and want to be taught/coached. That's at odds with 

> We welcome all experience levels, we also welcome first time contributors. 

---

Along similar lines, I'm not sure I agree with requiring DSF membership. That's again at odds with being open to all experience levels; DSF Members will, almost definitionally, have some experience already. 

I see contributing to the website as a really convenient "on ramp" to being involved in the Django community -- it's one of the few ways you could contribute to Django that involves merely writing a Django site, not doing something more complex. So I'd encourage keeping the requirements as minimal as possible.

Reply:
@jacobian seems most people would like individuals to be DSF members. But l total get your point, maybe we can refine this once we see the need to do so.

Reply:
> Could we set up a Google Form instead? From my experience with DSF mailing lists, it’s 99% spam that we receive on those emails.
> 
> It’d be nice for you to define here what kind of information people should provide when applying, rather than just expect applicants will figure it out.

We will be using PR for this.